crates/rpc-types-eth/src/block.rs

```diff
@@ -14,7 +15,7 @@ pub use alloy_eips::{
 /// Block representation
 #[derive(Clone, Debug, Default, PartialEq, Eq, Serialize, Deserialize)]
 #[serde(rename_all = "camelCase")]
-pub struct Block<T = Transaction> {
+pub struct Block {
```

Comment:
hmm, can we keep this generic? otherwise we can't reuse that block type?

Reply:
right, this could be useful in some cases, though I think for most of the rollups entire `Block` would be different as there'll be no withdrawals, so we'd likely want to just add more reusable generic components for custom blocks (like `BlockTransactions<T>`)

added generic back as it doesn't hurt anyway

Reply:
yep I also think this will be the case but this can be useful for rollups that only have 1 additional tx type for example